contracts/governance/test/ZeroExProtocolGovernor.t.sol

```diff
@@ -83,4 +90,58 @@ contract ZeroExProtocolGovernorTest is ZeroExGovernorBaseTest {
         state = governor.state(proposalId);
         assertEq(uint256(state), uint256(IGovernor.ProposalState.Executed));
     }
+
+    function testSecurityCouncilShouldBeAbleToExecuteRollback() public {
+        // Create a proposal
+        address[] memory targets = new address[](1);
+        targets[0] = address(zeroExMock);
+
+        uint256[] memory values = new uint256[](1);
+        values[0] = 0;
+
+        bytes[] memory calldatas = new bytes[](1);
+        bytes4 testFunctionSig = 0xc853c969;
+        address testFunctionImpl = 0x5615dEB798BB3E4dFa0139dFa1b3D433Cc23b72f;
+        calldatas[0] = abi.encodeWithSignature("rollback(bytes4,address)", testFunctionSig, testFunctionImpl);
+
+        // Security council adds the batch of rollbacks to the queue
+        vm.startPrank(securityCouncil);
+
+        bytes32 proposalId = timelock.hashOperationBatch(
+            targets,
+            values,
+            calldatas,
+            0,
+            keccak256(bytes("Emergency rollback"))
+        );
+        vm.expectEmit(false, false, false, false);
```

Comment:
Why ignore all the topics?

Reply:
Leftover from exclusion testing. My bad, reverted all to `true`.